group covariance mmrm (#97)
openpharma/mmrm: vignettes/introduction.Rmd

print(fit_cs)
```

## Grouped Covariance Structure

> Grouped covariance structures is supported by `mmrm` package. 

Reviewer: ```suggestion
Grouped covariance structures are supported by  the`mmrm` package. 
```

> Covariance matrix for each group are of identical structure(unstructured, compound symmetry, etc) but the values can be different.

Reviewer 2: Plural and slight rephrase

```suggestion
Covariance matrices for each group are identically structured (unstructured, compound symmetry, etc) but the estimates are allowed to vary across groups.
```

> We use the form `cs(time|group/subject)` to specify the group variable.

Reviewer 2: ```suggestion
We use the form `cs(time | group / subject)` to specify the group variable.
```

Here is an example of how we use `ARMCD` as group variable.

```{r group-cov}
fit_cs <- mmrm(
  formula = FEV1 ~ RACE + ARMCD * AVISIT + cs(AVISIT | ARMCD / USUBJID),
  data = fev_data,
  reml = FALSE
)

print(fit_cs$cov)
```

> We cam see that the covariance is different in different ARMCD group.

Reviewer: ```suggestion
We can see that the estimated covariance matrices are different in different `ARMCD` groups.
```
